sdks/nodejs/src/agonesSDK.js

```diff
@@ -24,8 +24,20 @@ class AgonesSDK {
 		this.emitters = [];
 	}
 
+	async connect() {
+		return new Promise((resolve, reject) => {
+			this.client.waitForReady(Date.now() + 30000, (error) => {
+				if (error) {
+					reject(error);
+				} else {
+					resolve();
+				}
+			})
+		});
+	}
+
 	async close() {
-		if (this.healthStream !== undefined) {
+		if (this.healthStream !== undefined){
```

Comment:
nit: you seem to have lost a space on this line

Reply:
ARG! Fixing.